aio_pika/robust_connection.py

```diff
@@ -124,7 +122,7 @@ def channel(self, channel_number: int = None,
         for number, channel in self._channels.items():
             try:
                 await channel.on_reconnect(self, number)
-            except (RuntimeError, ChannelClosed, AMQPError):
+            except (RuntimeError, ChannelClosed) + CONNECTION_EXCEPTIONS:
```

Comment:
I guess this is not so perfect. @mosquito what do you think?

Reply:
The global variable in module it's free I think. But creating a new tuple when preparing exception block isn't free.

Reply:
I returned changes back without `ChannelClosed` exception which inherits from `AMQPError`.